circle.yml

```diff
@@ -1,7 +1,16 @@
+dependencies:
+  override:
+    - make setup
+
 machine:
   node:
-    version: 0.10.28
+    version: 0.10.34
```

Comment:
No, though we do use one of the four tested versions.

There's no particular reason to change this line; I simply ran `cp ../sanctuary/circle.yml .`.